File: eWiLL_backend/src/main/kotlin/com/wipdev/eWiLL_backend/endpoints/payload/requests/Task.kt
package com.wipdev.eWiLL_backend.endpoints.payload.requests

import com.wipdev.eWiLL_backend.database.tables.course.Ruleset

data class Task(val id: Long?, val name: String?, val description: String?, val dueDate: String?, val subject: String?, val courseId:Long?,val solutionModelId: Long?,val rulesetId:Long?){



}

File: eWiLL_frontend/src/model/task/Task.ts
import Diagram from "../diagram/Diagram"

interface Task {
    id: number;
    name: string;
    description: string;
    dueDate: string
    mediaType: string 
    courseId: number;
    solutionModel: Diagram;
    rulsesetId: number;
    liability: string;
    
  }
  
  export default Task;